metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/SearchServiceImpl.java

```diff
@@ -151,7 +152,7 @@ public GroupResponse group(GroupRequest groupRequest) throws RestException {
       String sourceTypeField = Constants.SENSOR_TYPE.replace('.', ':');
```

Comment:
Just another point of duplication.  We define what the default value is here and in `ElasticsearchMetaAlertDao`.  Could be extracted out along with the other bits.

Reply:
This one is tricky.  We can't just reference ElasticsearchMetaAlertDao.SOURCE_TYPE because metron-elasticsearch is a runtime dependency in metron-rest.  The only other solution I can think of is to define a second constant in Constants but I think that is confusing since it's specific to ES.  I remember @justinleet working through this in another, maybe he can chime in.